packages/amesos2/src/KLU2/Include/klu2_ordinaltraits.h

```diff
@@ -112,28 +112,35 @@ struct KLU_OrdinalTraits<int>
 template<>
 struct KLU_OrdinalTraits<long int>
 {
-    static inline long int btf_order (long int n, long int *Ap, long int *Ai,
-        double maxwork, double *work, long int *P, long int *Q, long int *R, long int *nmatch,
-        long int *Work)
+// These should all be UF_long, which I presume is resolving to ptrdiff_t
+// ptrdiff_t is long int on Linux64, but just to be safe
+#ifdef _WIN32
```

Comment:
This is somewhat off. This is a specialization for "long int". Should be just have another one for ptrdiff_t ?

Reply:
Are you asking to dispense with kluint and just use ptrdiff_t?  I can make that change if that is what you want.

Reply:
I think I get it now.